server/neptune/workflows/internal/terraform/workflow.go

```diff
@@ -159,6 +161,43 @@ func (r *Runner) Plan(ctx workflow.Context, root *terraform.LocalRoot, serverURL
 	return response, nil
 }
 
+func (r *Runner) PolicyCheck(ctx workflow.Context, root *terraform.LocalRoot, serverURL *url.URL, showFile string) error {
```

Comment:
Alright hear me out lol, what if we started calling this something else. Something less ambiguous, like PlanValidation.

When this feature was built, I was reviewing all the PRs and didn't want to be super pedantic about the naming, but I figure since we're starting anew might as well bring it up. Like if you think about "PolicyCheck" what does that even mean, how would someone know what that does just by reading that.  Just my 2c, but if you don't want to change that name it's fine.

Reply:
Validate is also fine with me